docs/workspaces/people.md

```diff
@@ -32,3 +32,9 @@ To remove a user from the organization, select the options menu button
 ('three dots' button) to the right of the user and click **Remove**.  Note that **Org Owners** have implicit
 access to all workspaces in the organization, and you cannot revoke their access
 at the workspace level.
+
+## Last Activity
+
+The **Last Activity** column represents the most recent date on which an activity was performed by a user in a workspace. It serves as a useful indicator of user engagement, providing insights into workspace activity trends. The following actions currently count as user activity for a workspace:
```

Comment:
To be honest, I think your previous one was better - It was vague, but I dont like the bullets here, and its incomplete anyway. Let's use the previous one for now. I feel like we need to rework this whole page at some point anyway.